### ISSUE TITLE
[4.0] [backend-template] Question about toolbar buttons

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#28203]
### What needs to be fixed

Why is there so much space between the toolbar button icon and text?

![buttons](https://user-images.githubusercontent.com/2019801/75818618-17d99d00-5d91-11ea-91a4-92955fdcdabf.png)

If the space is reduced, there will be less chance of wrapping on smaller devices

**Comment by brianteeman** [comment]
hover on them and you will see why

**Comment by C-Lodder** [comment]
Yes, I know about the hover. Still doesn't explain the need for such a big space and why the icons need their own "container" (so to speak)

**Comment by N6REJ** [comment]
@C-Lodder looks like we can change the margin to `    margin: 0 0.25rem 0 -1.625rem;` and safely gain a lot of space back.

**Comment by infograf768** [comment]
Also, why lose all that space to the right of the text?
As a proof of concept, I did this patch (for LTR only) using existing variables.
It is even more important when the text is longer than en-GB
```
diff --git a/administrator/templates/atum/scss/blocks/_toolbar.scss b/administrator/templates/atum/scss/blocks/_toolbar.scss
index 24fe50a..14bcc87 100644
--- a/administrator/templates/atum/scss/blocks/_toolbar.scss
+++ b/administrator/templates/atum/scss/blocks/_toolbar.scss
@@ -53,5 +53,5 @@
 
   .btn {
-    padding: $input-btn-padding-y-sm-submenu $input-btn-padding-x-sm-submenu;
+    padding: $input-btn-padding-y-sm-submenu $badge-padding-x $input-btn-padding-y-sm-submenu $input-btn-padding-x-sm-submenu;
     font-size: 1rem;
     line-height: $atum-toolbar-line-height;
@@ -71,5 +71,5 @@
       width: 2.438rem;
       height: 100%;
-      margin: $input-btn-padding-y-sm-submenu $input-btn-submenu-icon-distance $input-btn-padding-y-sm-submenu -#{$input-btn-padding-x-sm-submenu};
+      margin: $input-btn-padding-y-sm-submenu $input-btn-padding-y-sm-submenu $input-btn-padding-y-sm-submenu -#{$input-btn-padding-x-sm-submenu};
       line-height: $atum-toolbar-line-height;
       color: var(--atum-text-dark);
```

### before patch
<img width="1474" alt="Screen Shot 2020-03-08 at 09 29 43" src="https://user-images.githubusercontent.com/869724/76159447-d1819680-6120-11ea-92a3-3250de3b68a2.png">

### after patch
<img width="1475" alt="Screen Shot 2020-03-08 at 09 34 32" src="https://user-images.githubusercontent.com/869724/76159457-e8c08400-6120-11ea-874a-fef9f4748b44.png">

**Comment by infograf768** [comment]
That patch would break the actions button, but easy to correct separately

**Comment by brianteeman** [comment]
Personally speaking it looks too busy and on small buttons like "help" it looks unbalanced. Also all other buttons would need to be updated as well such as "clear". The only real way to have a balanced view would be to abandon the two tone highlighting

**Comment by brianteeman** [comment]
??

![image](https://user-images.githubusercontent.com/1296369/77187873-08ff2400-6acd-11ea-9848-7270e32b9c8d.png)

**Comment by brianteeman** [comment]
This is fixed with #28607
